src/hotspot/cpu/arm/arm.ad

```diff
@@ -236,7 +236,8 @@ void MachConstantBaseNode::emit(CodeBuffer& cbuf, PhaseRegAlloc* ra_) const {
 
   Register r = as_Register(ra_->get_encode(this));
   CodeSection* consts_section = __ code()->consts();
-  int consts_size = consts_section->align_at_start(consts_section->size());
```

Comment:
-     int consts_size = consts_section->align_at_start(consts_section->size());
I have no idea what that means and:
+    // constants section size is aligned according to the align_at_start settings of the next section
And it is: int consts_size = CodeSection::align_at_Start(consts_section->size(), CodeBuffer::SECT_INSTS);
Still no idea.